File: software_challenge_sdk/src/framework/sc/shared/ScoreAggregation.java
package sc.shared;

import com.thoughtworks.xstream.annotations.XStreamAlias;

@XStreamAlias(value = "scoreAggregation")
public enum ScoreAggregation
{
	/**
	 * All values from all games should be summed up.
	 */
	SUM,

	/**
	 * All values from all games should be averaged.
	 */
	AVERAGE
}

File: sdk/src/main/framework/sc/shared/ScoreAggregation.kt
package sc.shared

import com.thoughtworks.xstream.annotations.XStreamAlias
import java.math.BigDecimal
import java.math.MathContext

/** How to aggregate the individual score values for final evaluation. */
@XStreamAlias(value = "scoreAggregation")
enum class ScoreAggregation {
    /** Sum the values of all games.  */
    SUM,
    
    /** Average the values of all games.  */
    AVERAGE;
}